Minor edits & paragraph of other open collaborative works

dhimmel opened the pull request:
Closes https://github.com/greenelab/meta-review/issues/25

The new paragraph is not necessarily in the right place. I did not make sure it flowed with the surrounding paragraphs. Would appreciate advice on where we think this paragraph should go. Could go intro, could go discussion.

dhimmel committed:
Change techblog citation to tag

Since URL was long

dhimmel committed:
Add paragraph on previous collaborative docs

Closes https://github.com/greenelab/meta-review/issues/25

agitter commented on content/02.main-text.md:
```diff
@@ -144,6 +144,19 @@ We annotated the author list to indicate that author order was partly randomized
 
+The Deep Review was not the first scholarly manuscript written online via an open collaborative process.
+Interestingly, the collaborative scholarly works (of which we're aware) tend to have dealt with rapidly developing topics.
+In 2013, two dozen mathematicians created the 600-page "HoTT Book" on "Homotopy Type Theory", writing collaboratively in LaTeX on GitHub [@url:https://homotopytypetheory.org/book/; @url:http://math.andrej.com/2013/06/20/the-hott-book/].
```
Unless you already checked in a local build, we should confirm after merging that the metadata was extracted correctly for these URL references.

agitter commented on content/02.main-text.md:
```diff
@@ -144,6 +144,19 @@ We annotated the author list to indicate that author order was partly randomized
+The Deep Review was not the first scholarly manuscript written online via an open collaborative process.
+Interestingly, the collaborative scholarly works (of which we're aware) tend to have dealt with rapidly developing topics.
+In 2013, two dozen mathematicians created the 600-page "HoTT Book" on "Homotopy Type Theory", writing collaboratively in LaTeX on GitHub [@url:https://homotopytypetheory.org/book/; @url:http://math.andrej.com/2013/06/20/the-hott-book/].
+More recently, Andreas Antonopoulos has led the writing of two technical books on cryptocurrency --- titled [Mastering Bitcoin](https://github.com/bitcoinbook/bitcoinbook) and [Mastering Ethereum](https://github.com/ethereumbook/ethereumbook) --- on GitHub in asciidoc format.
```
We could collapse these next three sentences.

> Two technical books on cryptocurrency --- [Mastering Bitcoin](https://github.com/bitcoinbook/bitcoinbook) and [Mastering Ethereum](https://github.com/ethereumbook/ethereumbook) --- written on GitHub in asciidoc format have engaged hundreds of contributors.

cgreene replied:
☝️ I agree with this collapsing

agitter commented on content/02.main-text.md:
```diff
@@ -144,6 +144,19 @@ We annotated the author list to indicate that author order was partly randomized
+The project lead then condensed this discussion into a more traditional journal commentary titled "Justify your alpha" with 88 authors [@doi:10.1038/s41562-018-0311-x].
+The greatest success to date of open collaborative writing is arguably Wikipedia, whose English version contains over 5.5 million articles.
+Wikipedia scaled encyclopedias [far beyond](https://en.wikipedia.org/wiki/Wikipedia:Size_comparisons) any privately-written alternative.
+As the examples above illustrate, open collaborative writing appears to scaling scholarly manuscripts where diverse opinion and expertise are paramount beyond what would otherwise be possible.
```
Should `appears` be `appeals`?

dhimmel replied:
I meant "appears to scale" rather than "appeals to scaling" but would be fine with either.

agitter replied:
`appears to scale` sounds good

agitter commented on content/02.main-text.md:
```diff
@@ -170,7 +183,7 @@ Conferences and open source projects have used codes of conduct to establish the
 We would encourage the maintainers of similar projects to consider broader codes of conduct for project participants that establish on social as well as academic norms.
 
 Open writing presents new opportunities for scholarly communication.
-`TODO: reference "paper of the future"? arXiv:1601.02927 doi:10.22541/au.149693987.70506124 doi:10.22541/au.148769949.92783646 http://blogs.nature.com/naturejobs/2017/06/01/techblog-c-titus-brown-predicting-the-paper-of-the-future`
```
Did you intentionally remove this other TechBlog article on the paper of the future?

dhimmel replied:
My bad, this change should be reverted. I didn't realize this wasn't a cite for the recent techblog on deep review. Good example of diffs help catch stupid errors!

cgreene replied:
#meta

agitter reviewed (commented):
Looks good overall and adds important context.  We could condense the text even further in a few places if we want.

I agree that this should go somewhere in the Discussion.  I'd like to keep the Intro short so we can get to our contributions quickly.

cgreene reviewed (approved):
I agree with @agitter's point about condensing, but largely LGTM. Happy to see it merged after the sentences are condensed and @agitter approves.

agitter commented on content/02.main-text.md:
```diff
@@ -144,6 +144,19 @@ We annotated the author list to indicate that author order was partly randomized
+Both books have engaged hundreds of contributors.
+Mastering Ethereum has yet to be published, while Mastering Bitcoin continues to be maintained and expanded since its 2014 publication by O'Reilly Media.
+A 2017 perspective on the future of peer review was written collaboratively on Overleaf, with contributions from 32 authors [@doi:10.12688/f1000research.12037.2].
+Also in 2017, while debate was raging over tightening the default threshold for statistical significance, nearly 150 scientists contributed to a Google Doc, which reached 100-pages in length [@doi:10.1126/science.aat0471].
```
An option for condensing these two lines:
> While debate was raging over tightening the default threshold for statistical significance, nearly 150 scientists contributed to a Google Doc discussion [@doi:10.1126/science.aat0471] that was condensed into a traditional journal commentary [@doi:10.1038/s41562-018-0311-x].

Or the even shorter:
> Nearly 150 scientists contributed to a Google Doc discussion about statistical significance thresholds [@doi:10.1126/science.aat0471] that was condensed into a traditional journal commentary [@doi:10.1038/s41562-018-0311-x].

dhimmel committed:
Update published arXiv citations

dhimmel committed:
Address reviewer comments

dhimmel committed:
Update manual references

dhimmel commented:
As of https://github.com/greenelab/meta-review/pull/38/commits/4220837410097f965b6ec4609a71e23991ad145b, local build [manuscript.pdf](https://github.com/greenelab/meta-review/files/1835534/manuscript.pdf)

agitter commented:
The updates look good.  I noticed two author extraction problems in the pdf.

- Editor Jack Leeming is shown as the author of the Manubot TechBlog reference [2] instead of Jeffrey Perkel.
- Reference [27] shows Jop Vrieze instead of Jop de Vrieze as the author.  I'm not positive, but I expect that `de Vrieze` is the family name.

dhimmel commented:
Regarding "de Vrieze", this is an upstream problem. I believe the issue is that _Science_ improperly deposited the [metadata](https://search.crossref.org/?q=10.1126%2Fscience.aat0471) to Crossref. This is the CSL returned from Crossref:

```yml
    "author": [
      {
        "given": "Jop",
        "family": "Vrieze",
        "affiliation": []
      }
    ],
```

I've reported the [issue on Twitter](https://twitter.com/dhimmel/status/977181539254652929). I don't think we should override this. Journals need to start properly submitted metadata or else expect for citations of their articles to be messed up.

dhimmel committed:
Manual metadata for Manubot TechBlog

Addresses:

> Editor Jack Leeming is shown as the author of the Manubot
TechBlog reference [2] instead of Jeffrey Perkel.

agitter reviewed (approved):
I agree that [27] should be fixed upstream.

agitter commented:
@dhimmel merge whenever you're ready.
